development/jobguard/README.md

```diff
@@ -11,24 +11,30 @@ The main purpose of Job Guard is to delay running integration jobs that depend o
 
 To run the application, use this command:
 
-```bash
-COMMIT_SHA={commit_sha} PROW_CONFIG_FILE={prow_config_file} PROW_JOBS_DIRECTORY={prow_jobs_directory} go run cmd/main.go
+|Flag|Required|Description|
+|---|---|---|
+|`-github-host`| No | GitHub's default host.|
+|`-github-endpoint`| No | GitHub API endpoint.|
+|`-github-graphql-endpoint`| No | GitHub GraphQL API endpoint.|
+|`-github-token-path`|Yes|Path to the file containing the GitHub OAuth secret.|
+|`-debug`|No|Enable debug logging.|
+|`-dry-run`|No|Run in dry mode.|
+|`-expected-contexts-regexp`|Yes|Regular expression with expected contexts.|
+|`-fail-on-no-contexts`|No|Fail if regexp does not match to any of the GitHub contexts.|
```

Comment:
```suggestion
|`-fail-on-no-contexts`|No|Fail if regexp does not match any of the GitHub contexts.|
```
Use `match` in this way: `A matches B` or `You match A to B` or `A is a match for B`.